releases/v0.3.0/EVALUATION_DEPLOYMENT.md

```diff
@@ -0,0 +1,186 @@
+# CloudStack Cluster API Provider (CAPC) Release v.0.3.0 Evaluation Deployment Guide
+
+The CloudStack Cluster API Provider (CAPC) v0.3.0 is a pre-release of partial CAPC functionality for the purposes
+of stakeholder review.  As an interim release, the process for deploying it is still under development.  This document
+defines a manual deployment process suitable for evaluating this CAPC interim release.
+
+## Evaluation Environment Pre-Requisites:
+
+### - A running Kubernetes cluster for hosting CAPC
+
+This should be an easily disposable/re-creatable cluster, such as a locally-running kind (Kuberetes in Docker) cluster.
+
+Your KUBECONFIG file's *current-context* must be set to the cluster you want to use.
+
+### - CAPI clusterctl v0.3.24 (https://github.com/kubernetes-sigs/cluster-api/releases/tag/v0.3.24)
+
+The currently released CAPI clusterctl 1.x is not compatible with this pre-release.
+
+### - A CloudStack Environment
+- Zone
+- Network
+- CAPI-compatible QEMU template (i.e., created with https://github.com/kubernetes-sigs/image-builder)
+- Machine Offerings (suitable for running Kubernetes nodes)
+- apikey and secretkey for a CloudStack user having domain administrative privileges
+- k8s Control Plane IP Address (Shared network: available IP address in the network range; isolated network: public IP address)
+
+## Deployment Steps
+### Define Identity Environment Variable
+
+An environment variable named CLOUDSTACK_B64ENCODED_SECRET must be defined, containing the base64 encoding of a 
+cloud-config properties file.  This file is of the form:
+
+```
+[Global]
+api-url = <urlOfCloudStackAPI>
+api-key = <cloudstackUserApiKey>
+secret-key = <cloudstackUserSecretKey>
+```
+After defining this in a file named cloud-config, create the environment variable with:
+
+```
+export CLOUDSTACK_B64ENCODED_SECRET=$(cat cloud-config | base64 | tr -d '\n')
```

Comment:
The base64 executable in some Linux distributions wraps lines by default, which causes CLOUDSTACK_B64ENCODED_SECRET to be a multi-line string. Adding a step to execute `echo $CLOUDSTACK_B64ENCODED_SECRET` may be a good idea to make sure the environment variable is a single-line string. In the case that the string is a multi-line string, adding a -w0 flag to base64 would disable the default line wrapping.

Reply:
Thanks @wongni, I'll create a ticket to do this for 0.3.1.